configure.ac

```diff
@@ -4750,6 +4750,12 @@ elif test "$disable_gil" = "yes"; then
   AC_MSG_ERROR([--disable-gil requires mimalloc memory allocator (--with-mimalloc).])
 fi
 
+if test "$with_mimalloc" = yes; then
+  if test "$with_tsan" = yes; then
+    AC_DEFINE([MI_TSAN], [1], [Define MI_TSAN for mimalloc thread sanitizer support])
```

Comment:
I think we should do this in `pycore_mimalloc.h` instead. We already do that for `MI_DEBUG` and some other defines.

Something like:


```
#ifdef _Py_THREAD_SANITIZER
#  define MI_TSAN 1
#endif
```

Reply:
Hm, yes that looks like a better solution.

Reply:
Done :)